### PR TITLE
RELATED: RAIL-2825 Propagate user saved separators & other relevant settings in DashboardView

### PR DESCRIPTION
**Pull request by kandl**
- Reflect user saved regional number formatting (ISeparators) in DashboardView
- Reflect disableKpiDashboardHeadlineUnderline feature flag in DashboardView
- Minor cleanup and bug fixes

JIRA: RAIL-2825

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: libs/sdk-ui-ext/src/internal/dashboardEmbedding/KpiView/KpiRenderer.tsx [hunk 1]
```diff
@@ -72,7 +74,7 @@ export const KpiRenderer: React.FC<IKpiRendererProps> = ({
                 </div>
                 <ResponsiveText>
                     <div className="headline-value-wrapper" onClick={onPrimaryValueClick}>
-                        {primaryValue?.isDrillable ? (
+                        {primaryValue?.isDrillable && !disableDrillUnderline ? (
```
Comment (no23reason) [inline libs/sdk-ui-ext/src/internal/dashboardEmbedding/KpiView/KpiRenderer.tsx:77]: This is not so simple, even with `disableDrillUnderline` it should be rendered as link, just without the "headline-link-style-underline" class (see TODO on line 30).

File: libs/sdk-ui/src/index.ts [hunk 2]
```diff
@@ -6,8 +6,4 @@ export * from "./base";
 export * from "./execution";
 export * from "./kpi";
 
-/**
```
Comment (no23reason) [inline libs/sdk-ui/src/index.ts:9]: Should we keep the TSDoc (at least with the public spec)?

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/585/ : SUCCESS in 26s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/846/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3445/ : FAILURE in 14m 08s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/586/ : SUCCESS in 29s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/847/ : SUCCESS in 26s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3448/ : SUCCESS in 14m 47s

**Pull request merged**